dartpad_codelabs/src/getting_started_with_slivers/step_06/solution.dart

```diff
@@ -31,7 +31,10 @@ class HorizonsApp extends StatelessWidget {
                 background: Stack(
                   fit: StackFit.expand,
                   children: <Widget>[
-                    Image.asset(baseAssetURL + 'assets/header.jpeg', fit: BoxFit.cover),
+                    Image.network(
+                      baseAssetURL + 'assets/header.jpeg',
```

Comment:
nit: 
```suggestion
                      '${baseAssetUrl}assets/header.jpeg',
```

https://dart.dev/guides/language/effective-dart/usage#prefer-using-interpolation-to-compose-strings-and-values